pkgs/by-name/cs/cssmodules-language-server/package.nix

```diff
@@ -0,0 +1,33 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+}:
+
+let
+  name = "cssmodules-language-server";
+  url = "https://github.com/antonk52/${name}";
+in
+buildNpmPackage (finalAttrs: {
+  pname = name;
+  version = "1.5.2";
+
+  src = fetchFromGitHub {
+    owner = "antonk52";
+    repo = name;
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-9RZNXdmBP4OK7k/0LuuvqxYGG2fESYTCFNCkAWZQapk=";
+  };
+
+  npmDepsHash = "sha256-1CnCgut0Knf97+YHVJGUZqnRId/BwHw+jH1YPIrDPCA=";
+
+  meta = with lib; {
```

Comment:
Please move this `with lib;` to only `maintainers = with lib.maintainers; [ ... ]`